### ISSUE TITLE
Can you describe a bit more what this does ?

### ISSUE THREAD

**Issue by X-Ryl669** [skinofthesoul/grav-plugin-automagic-images#1]
First, I'll like to thank you for your plugin, I think it's currently missing from Grav.
Can you describe a bit more what this does ?

As far as I've understand, let's say I'm uploading a 3000x2000 picture in the admin page I'm editing.

What will happen next ?
Will it create a 1500x1000 picture ?
A 750x500 picture ? (and so on)

Will it change the output HTML to `<img src=bigPic srcset="smallerPic smallestPic" sizes="???">` and if yes, what is set in the srcset and sizes attribute ? Is it configurable ?

Also, if we use `?lightbox&resize=500` in markdown, is it accounted for ?

Can we use webp pictures ?

**Comment by skinofthesoul** [comment]
Excellent questions, thank you! I have updated the README and would love your feedback on that. Is everything clear now?

Also I have just checked what happens for webp pictures, and they are not supported right now, only jpg and png. However, looking at the relevant bits of code which are still from the original Resize Images plugin, I think it should be fairly simple to add support for other image formats. I'll put it on my list and see when I can get around to that. :-)
